resources/scripts/ttx_diff.py

```diff
@@ -616,6 +610,19 @@ def delete_things_we_must_rebuild(rebuild: str, fontmake_ttf: Path, fontc_ttf: P
                     os.remove(path)
 
 
+# returns the path to the compiled binary
+def build_crate(manifest_path: Path):
```

Comment:
We already have logic to find the root fontc directory. We already assume the paths of crates. It would be more consistent with the existing code to just presume you'll find the binary at root / "target" / "release" / the_name.